dune

```
@@ -0,0 +1,18 @@
+(library
+ (public_name zip)
```

```suggestion
 (public_name camlzip)
```

Another diff that I think is necessary for a drop-in replacement. (And there's an existing `camlzip.opam` file that looks unnecessary now.)

---

`dose`, `opam` and `ocamlnet` might be affected by such change. It is not clear from a quick grep why opam.rpm depends on this pkg, this needs more investigation.

Anyway, any API change that may be desirable should probably be done in a separate submission.

---

Isn't the opposite the case? Tezos is currently consuming this package under the `camlzip` name (https://gitlab.com/tezos/tezos/-/blob/master/src/lib_store/dune#L14), and the opam package is released as `camlzip`.